cedars/app/db.py

```diff
@@ -496,12 +496,13 @@ def get_documents_to_annotate(patient_id=None):
     return documents_to_annotate
 
 
-def get_all_annotations_for_patient(patient_id):
+def get_all_annotations_for_patient(patient_id, unique_notes = True):
```

Comment:
We need to see all sentences in a note. However, 2 use cases need to be handled:
1. If a sentence has multiple matching keywords, we should show the sentence just once
2. Add `Skip Duplicates` - not showing exact same sentences - just show the first one sorted by time